### PR TITLE
fix: add missing goldfile for diff DBs

### PR DESCRIPTION
**Pull request by jainkaran98**
Updated test data and goldfiles.

### CODE REVIEW

**Comment by jainkaran98** [comment]
@dgyawali @sahil-here Here is the PR for checking-in missing goldfiles. Can you please merge.

File: impl/src/test/goldfiles/FormulaFields/testBeginsFunction.xml
Comment (sahil-here) [inline impl/src/test/goldfiles/FormulaFields/testBeginsFunction.xml:1]: I don't see this testcase in current formulatests.xml or formulatests-math.xml. Let me know if I am looking at an older version of it.

File: impl/src/test/goldfiles/FormulaFields/testEndsFunction2.xml
Comment (sahil-here) [inline impl/src/test/goldfiles/FormulaFields/testEndsFunction2.xml:1]: I don't see this testcase in current formulatests.xml or formulatests-math.xml. Let me know if I am looking at an older version of it.

File: impl/src/test/goldfiles/FormulaFields/testEndsFunction3.xml
Comment (sahil-here) [inline impl/src/test/goldfiles/FormulaFields/testEndsFunction3.xml:1]: I don't see this testcase in current formulatests.xml or formulatests-math.xml. Let me know if I am looking at an older version of it.

Comment (jainkaran98) [reply impl/src/test/goldfiles/FormulaFields/testEndsFunction3.xml:1]: Thanks @sahil-here for pointing this out, i was doing a POC in local long back and these files must have somehow got added in this review mistakenly. Strange that workflow is passing given that ENDS function is not even exposed.

Have updated the review.

**Pull request merged**